tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt

```diff
@@ -243,6 +243,7 @@ Google	;	yo	;	modern	;	Yoruba
 Google	;	yue	;	modern	;	T5	Cantonese
 Google	;	zh	;	modern	;	T1	Simplified Chinese
 Google	;	zh_Hant	;	modern	;	T1	Traditional Chinese
+Google  ; zh_Hant  ; modern 
```

Comment:
Don't you mean to add `zh_Hant_HK` here?

Reply:
argh